### PR TITLE
CB-11477 Add a page about nightly builds

### PR DESCRIPTION
**Pull request by vladimir-kotikov**
The PR adds a separate page under `contributing` directory with the information about nightly builds usage

### CODE REVIEW

**Comment by vladimir-kotikov** [comment]
@nikhilkh, @stevengill, could you please take a look?

**Comment by vladimir-kotikov** [comment]
@stevengill, ping

File: www/contribute/index.html [hunk 1]
```diff
@@ -22,6 +22,7 @@
             <li>Join the <a href="{{ site.baseurl }}/contact">Dev mailing list</a> and send a brief introduction of yourself to it</li>
             <li>Join the discussion on <a href="http://slack.cordova.io/">Slack</a></li>
             <li>Sign the <a href="http://www.apache.org/licenses/#clas">Individual Contributor License Agreement (ICLA)</a> and <a href="mailto:secretary@apache.org">submit it</a></li>
+            <li>Try out the next version of Cordova using <a href="./nightly_builds.html">nightly builds</a></li>
```
Comment (dblotsky) [inline www/contribute/index.html]: Maybe use an absolute link like `{{ site.baseurl }}/contribute/nightly_builds.html` here, in case this page ever moves.

**Comment by stevengill** [comment]
LGTM. Want me to merge it in? Maybe make the change @dblotsky recommended.

**Comment by vladimir-kotikov** [comment]
Updated and merged. Thanks, @dblotsky, @stevengill

**Pull request merged**